Bryn Mawr Classical Review.js

```diff
@@ -59,6 +76,8 @@ function doWeb(doc, url) {
 	Zotero.Utilities.processDocuments(arts, function(doc) {
```

Comment:
Ideally, the whole translator should be slightly restructed with  `Zotero.selectItems` written as a callback function and the part below this as a separate `scrape` function. Will work better in all kinds of ways.
See https://github.com/zotero/translators/blob/master/Digital%20Medievalist.js#L25 for an example. Let me know if that's unclear.